Dockerfile

```diff
@@ -1,22 +1,54 @@
-FROM ubuntu:16.04
-
-# Install ansible and awscli
-RUN apt-get update \
-      && apt-get install --no-install-recommends --no-install-suggests -y \
-      python-dev \
-      python-pip \
-      gcc \
-      libssl-dev \
-      curl \
-      groff \
-      && pip install --upgrade pip setuptools \
-      && pip install netaddr awscli ansible Jinja2 \
-      && rm -rf /var/lib/apt/lists/*
+FROM alpine:3.6
+ENV ANSIBLE_VERSION 2.4.1.0
+ENV JINJA2_VERSION 2.10
+ENV AWSCLI_VERSION 1.11.185
+ENV KUBE_AWS_RELEASE v0.9.8
+ENV KUBECTL_VERSION v1.8.5
+
+RUN apk update \
+    && apk add --no-cache unzip curl tar bash \
+    python make bash vim jq  \
+    openssl openssh-client sshpass  \
+    gcc libffi-dev python-dev musl-dev openssl-dev py-pip py-virtualenv \
```

Comment:
`gcc` need only for packages building, so it can be removed below

Reply:
Python packages installation reqires `gcc`

Reply:
it can be removed after installation of packages:
```
RUN apk add gcc 
RUN pip something 
RUN apk delete gcc
```

Reply:
nvm difference in ~5mb

Reply:
It won't bring much benefits.